SEOImages: User Story 1463880, Part 21

damabe opened the pull request:
## Summary

Updating image file names and references to improve SEO.

Files #1463880

Contributes to #10839

damabe committed:
Image updates.

damabe committed:
Image updates.

damabe committed:
Merge branch 'master' of https://github.com/dotnet/docs

damabe committed:
Merge branch 'master' of https://github.com/dotnet/docs

damabe committed:
Merge branch 'master' of https://github.com/dotnet/docs

damabe committed:
Merge branch 'master' of https://github.com/dotnet/docs

damabe committed:
Merge branch 'master' of https://github.com/dotnet/docs

damabe committed:
Merge branch 'master' of https://github.com/dotnet/docs

damabe committed:
Merge branch 'master' of https://github.com/dotnet/docs

damabe committed:
SEO image updates.

damabe committed:
SEO image updates.

damabe committed:
SEO image updates.

damabe committed:
SEO image updates.

damabe committed:
SEO image updates.

damabe committed:
SEO image updates.

damabe committed:
SEO image updates.

damabe commented:
@mairaw This PR is ready for review. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019031700036/Report/
There are five image names identified by CATS as being too short. They should be handled at a later time as they are not part of the SEO image without hyphen work.

mairaw commented on docs/core/tutorials/debugging-with-visual-studio.md:
```diff
@@ -33,7 +33,7 @@ Run your program in Debug mode and try a few debugging features:
 1. Program execution stops when it reaches the breakpoint and before the `Console.WriteLine` method executes. The **Autos** window displays the values of variables that are used around the current line. The **Locals** window (which you can view by clicking the **Locals** tab) displays the values of variables that are defined in the currently executing method.
 
-   ![Visual Studio application window](./media/debugging-with-visual-studio/break.png)
+   ![Screenshot that shows an example of a breakpoint.](./media/debugging-with-visual-studio/breakpoint-console-window.png)
```
```suggestion
   ![Screenshot of a breakpoint in Visual Studio.](./media/debugging-with-visual-studio/breakpoint-console-window.png)
```

mairaw commented on docs/framework/security/wif-claims-programming-model.md:
```diff
@@ -7,7 +7,7 @@ author: "BrucePerlerMS"
 ASP.NET and Windows Communication Foundation (WCF) developers ordinarily use the IIdentity and IPrincipal interfaces to work with the user’s identity information. In .NET 4.5, Windows Identity Foundation (WIF) has been integrated such that claims are now always present for any principal as illustrated in the following diagram:
 
- ![WIF Claims Programming Model](../../../docs/framework/security/media/wifclaimsprogrammingmodel.png "WIFClaimsProgrammingModel")
+ ![Diagram that shows the WIF Claims Programming Model.](./media/wif-claims-programming-model/wif-claims-programming-model.png)
```
```suggestion
 ![Diagram that shows the WIF Claims programming model.](./media/wif-claims-programming-model/wif-claims-programming-model.png)
```

mairaw commented on docs/framework/wcf/feature-details/message-security-with-a-certificate-client.md:
```diff
@@ -11,7 +11,7 @@ The following scenario shows a Windows Communication Foundation (WCF) client and
  For a sample application, see [Message Security Certificate](../../../../docs/framework/wcf/samples/message-security-certificate.md).  
   
- ![Client with certificate](../../../../docs/framework/wcf/feature-details/media/clientwithcertificate.gif "ClientWithCertificate")  
+ ![Screenshot that shows a client with certificate.](./media/message-security-with-a-certificate-client/client-with-certificate.gif)  
```
Can you move this before the sentence in line 12?

mairaw commented on docs/framework/windows-workflow-foundation/how-to-create-a-sequential-workflow.md:
```diff
@@ -141,9 +141,9 @@ Workflows can be constructed from built-in activities as well as from custom act
+     The following example illustrates the completed workflow:  
   
-     ![Completed Sequential Workflow](./media/wfsequentialgettingstartedtutorialcomplete.JPG "WFSequentialGettingStartedTutorialComplete")  
+     ![Screenshot that shows a completed sequential workflow.](./media/how-to-create-a-sequential-workflow/complete-sequential-workflow.jpg)  
```
```suggestion
     ![Screenshot that shows the completed sequential workflow.](./media/how-to-create-a-sequential-workflow/complete-sequential-workflow.jpg)  
```

mairaw commented on docs/framework/winforms/advanced/how-to-draw-opaque-and-semitransparent-lines.md:
```diff
@@ -19,9 +19,9 @@ When you draw a line, you must pass a <xref:System.Drawing.Pen> object to the <x
  The following example draws a bitmap and then draws three lines that use the bitmap as a background. The first line uses an alpha component of 255, so it is opaque. The second and third lines use an alpha component of 128, so they are semitransparent; you can see the background image through the lines. The statement that sets the <xref:System.Drawing.Graphics.CompositingQuality%2A> property caus ...
   
- The following illustration shows the output of the following code.  
+ The following illustration shows the output of the following code:  
```
can you move this to after the sample code?

mairaw commented on docs/framework/winforms/advanced/how-to-draw-with-opaque-and-semitransparent-brushes.md:
```diff
@@ -18,10 +18,10 @@ When you fill a shape, you must pass a <xref:System.Drawing.Brush> object to one
 ## Example  
  The following example draws a bitmap and then fills three ellipses that overlap the bitmap. The first ellipse uses an alpha component of 255, so it is opaque. The second and third ellipses use an alpha component of 128, so they are semitransparent; you can see the background image through the ellipses. The call that sets the <xref:System.Drawing.Graphics.CompositingQuality%2A> property causes th ...
+
+ The following illustration shows the output of the following code: 
```
same here. can you move this to after the code?

mairaw commented on docs/framework/winforms/advanced/how-to-draw-with-opaque-and-semitransparent-brushes.md:
```diff
@@ -18,10 +18,10 @@ When you fill a shape, you must pass a <xref:System.Drawing.Brush> object to one
- The following illustration shows the output of the following code.  
-  
- ![Opaque and Semitransparent](./media/compqualellipse.png "compqualellipse")  
+ ![Illustration that shows opaque and semitransparent output.](./media/how-to-draw-with-opaque-and-semitransparent-brushes/comp-qual-ellipse-semitransparent.png)  
```
maybe use the full name of the property?

```suggestion
 ![Illustration that shows opaque and semitransparent output.](./media/how-to-draw-with-opaque-and-semitransparent-brushes/compositingquality-ellipse-semitransparent.png)  
```

mairaw commented on docs/framework/winforms/advanced/media/how-to-draw-with-opaque-and-semitransparent-brushes/desktop.ini:
```diff
@@ -0,0 +1,2 @@
+[LocalizedFileNames]
```
we shouldn't be adding this file, right?

mairaw commented on docs/standard/data/xml/xslt-transformations-with-the-xsltransform-class.md:
```diff
@@ -18,8 +18,7 @@ The goal of the XSLT is to transform the content of a source XML document into a
 
-![XSLT Transformation Architecture](media/xslttransformationswithxsltransformclass.gif "xsltTransformationsWithXslTransformClass")  
-Transformation Architecture
+![Diagram that shows the XSLT Transformation Architecture.](./media/xslt-transformations-with-the-xsltransform-class/xslt-transformation-architecture.gif) 
```
```suggestion
![Diagram that shows the XSLT transformation architecture.](./media/xslt-transformations-with-the-xsltransform-class/xslt-transformation-architecture.gif) 
```

mairaw reviewed (changes requested):
Thanks @damabe. A few comments before we merge,

mairaw commented:
@damabe can you address my suggestions here?

damabe committed:
Reviewer feedback.

damabe committed:
Reviewer feedback.

damabe commented:
@mairaw I've addressed your requested changes. Please have a look. Updated CATS: https://contentqacats.azurewebsites.net/CheckResults/2019032300033/Report/

There are five image names identified by CATS as being too short. They should be handled at a later time as they are not part of the SEO image without hyphen work.

mairaw reviewed (approved):
Thanks @damabe!
